src/sentry/static/sentry/app/views/settings/projectPlugins/index.tsx

```diff
@@ -65,7 +65,6 @@ class ProjectPluginsContainer extends React.Component<Props> {
 
         <ProjectPlugins
           {...this.props}
-          onError={this.fetchData}
```

Comment:
There is no onError for `ProjectPlugins`. Again `propTypes` were masking this value missing.